src/main/java/HomePage.java

```diff
@@ -0,0 +1,23 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class HomePage {
+    WebDriver driver;
+    By ResearchBarSelector = By.id("twotabsearchtextbox");
+    By ButtonSearchSelector = By.id("nav-search-submit-button");
+    public HomePage(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public HomePage enterTextSearchBar(String text){
+
+        driver.findElement(ResearchBarSelector).sendKeys(text);
+        return this;
+    }
+    public SearchResultPage OpenSearchResultPage(){
```

Comment:
can you merge this function with enterTextSearchBar? create a new function:
`public SearchResultPage search(String text)` 
which not only enter the search but which also click the button